config/development.json

```diff
@@ -4,6 +4,7 @@
 		"max_static_paths": 0
 	},
 	"dev_dot": {
+		"beta_product_slugs": ["hcp", "nomad", "vault", "waypoint"],
```

Comment:
We'll have to think of a workaround....

Reply:
Also, I think you would just want to update `config/base.json` since preview and developer extend the base. Then production overrides that base setting.